src/lib.rs

```diff
@@ -392,6 +393,14 @@ impl<'a> DeployData<'a> {
         };
         Ok(profile_user)
     }
+
+    fn get_sudo(&'a self) -> Result<String, DeployDataDefsError> {
+       let sudo = match self.merged_settings.sudo {
+           Some(ref x) => x.clone(),
+           None => "sudo -u".to_string()
+        };
+        Ok(sudo)
+    }
```

Comment:
Does it make sense to provide a default value here, or are defaults hidden somewhere else?

Reply:
This looks fine to me :)